### PR TITLE
Add fixes for Nextcloud-related webapps

### PR DESCRIPTION
**Pull request by csolisr**
This commit fixes the parsing algorithm for the website to use `safeParseInt` across all Nextcloud applications, modifies the default items on the settings dropdown menu to be better localized to each individual page, and corrects the format of the URL to better accommodate the custom servers used by Nextcloud.

### CODE REVIEW

File: recipes/nextcloud-talk/webview.js
Comment (csolisr) [inline recipes/nextcloud-talk/webview.js:15]: Please wait until PR #64 is merged. This line needs to be corrected accordingly.

**Comment by vraravam** [comment]
@csolisr - please go ahead, I have merged #64

File: recipes/nextcloud-carnet/package.json
Comment (SpecialAro) [inline recipes/nextcloud-carnet/package.json:4]: The version bump should be `1.2.1` and not `1.3.0`

File: recipes/nextcloud-talk/webview.js
Comment (csolisr) [reply recipes/nextcloud-talk/webview.js]: Merge solved

File: recipes/nextcloud-carnet/package.json
Comment (csolisr) [reply recipes/nextcloud-carnet/package.json]: Thanks for the clarification, done

**Review by vraravam — CHANGES_REQUESTED** [review]

File: recipes/nextcloud-talk/package.json
Comment (csolisr) [reply recipes/nextcloud-talk/package.json]: Done, please review

**Review by vraravam — APPROVED** [review]

**Comment by vraravam** [comment]
@all-contributors please add @csolisr for code

**Pull request merged**
